### DISCUSSION TITLE
import/image missing in new-backend?

### DISCUSSION THREAD

**Discussion by dennisbergstrom** [hajkmap/Hajk#1007] [Q&A]
Hi!

I am trying out hajkMap and have added a geoJson layer to my map successfully, but i need the points in my layer to have a beautiful icon, but I am unable to upload an icon in new_admin. The controller "import" seems to be missing in new_backend? Or am I misunderstanding something?

**Comment by jacobwod** [comment]
Admin UI currently lacks the upload functionality - the file selection dialog is slightly misleading in that way. What you need to do is to upload the icon to a directory on a publicly available server and point Hajk into the icon. You can use absolute or relative path.

**Reply by bitmapbulgaria** [reply]
Just try to help:
It is possible to make an /img folder on your own server and just configure your web server to proxy to this folder, when <yourserverURL>/img is called from the app. 
Another good (and simple) idea is just to create folder in (for example:  /usr/share/geoserver/webapps/img ) and Jetty will take care to make it available to the web on <yourServerURL>:8080/img

This will save you effort for create ftp or cloud endpoint for serving some simple icons, though.

**Reply by jacobwod** [reply]
I agree @bitmapbulgaria. That's the solution I've seen in most installations: you expose a static folder via either Hajk's Backend's proxy functionality or, in case you have another web server in front of Hajk (such as an NGINX proxy) you let it expose the static images/documents.
